runtime/common/src/lib.rs

```diff
@@ -132,7 +133,7 @@ parameter_types! {
 	/// Minimum amount of the multiplier. This value cannot be too low. A test case should ensure
 	/// that combined with `AdjustmentVariable`, we can recover from the minimum.
 	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000_000u128);
-	pub const MultiPaymentCurrencySetFee: Pays = Pays::Yes;
+	pub MaximumMultiplier: Multiplier = Bounded::max_value();
```

Comment:
shouldn't be this set to less than impossible to pay value? so it is still somewhat possible to interact with chain and not completely out of reach to anyone? especially when "free" extrinsic that still take weight can drive the price up

Reply:
I compared the implementations and this should behave the same way and provide the same results as before. Before, the returned value was calculated as `something.max(min_multiplier)`. Now the value is clamped `something.clamp(min_multiplier, max_multiplier)`. So the new param `MaximumMultiplier` is used to restrict the multiplier, which was "unlimited" before.

Reply:
i understand that it was unlimited before - it was definitely an issue. that is why I want to utilize this param actually and set it sensibly

Reply:
lets fix it after merge in https://github.com/galacticcouncil/Basilisk-node/issues/613